prowler/lib/outputs/outputs.py

```diff
@@ -8,7 +8,7 @@ def stdout_report(finding, color, verbose, status, fix):
     if finding.check_metadata.Provider == "aws":
         details = finding.region
     if finding.check_metadata.Provider == "azure":
-        details = finding.check_metadata.ServiceName
+        details = finding.location
```

Comment:
Why this?

Reply:
because we were using ServiceName since we didn't manage locations for Azure